--- /dev/null
+++ ../src/Nacmartin/PhpExecJs/RuntimeAutodetector.php
@@ -14,7 +14,7 @@
         // Runners will be checked for availabilty by order in array so order
         // matters
         $this->runtimes[] = new V8jsRuntime('V8js PHP Extension (V8)');
-        $this->runtimes[] = new ExternalRuntime('Node.js (V8)', array('node', 'nodejs'));
+        $this->runtimes[] = new ExternalRuntime('Node.js (V8)', array('node', 'nodejs', 'nodejs14'));
     }
 
     public function autodetect()
